JobConfig/common/mu2eg4.fcl

```diff
@@ -22,13 +22,168 @@ mu2e.physics.producers.g4run.commonBase: {
    Mu2eG4CommonCut: {}
 }
 
+
```

Comment:
I am unsure if these parts belong in mu2eg4 or cosmic/prolog.  I believe they are only used in comsic, so cosmic/prolog is probably better.